src/core/load-fragment-gperf.gperf.in

```diff
@@ -86,6 +86,7 @@
 {{type}}.SystemCallLog,                       config_parse_warn_compat,                           DISABLED_CONFIGURATION,             0
 {{type}}.MemoryDenyWriteExecute,              config_parse_warn_compat,                           DISABLED_CONFIGURATION,             0
 {{type}}.RestrictNamespaces,                  config_parse_warn_compat,                           DISABLED_CONFIGURATION,             0
+{{type}}.DelegateNamespaces,                  config_parse_warn_compat,                           DISABLED_CONFIGURATION,             0
```

Comment:
This is wrong. DelegateNamespaces= doesn't depend on seccomp at all. Please move it out of the if block instead.

Reply:
I'm happy with that.

Reply:
Done @YHNdnzj